addons/l10n_ug/data/account_tax_report_data.xml

```diff
@@ -514,21 +514,49 @@
                             <record id="F31_normal_tax" model="account.report.expression">
                                 <field name="label">hidden_computation_of_F31_using_normal_formula</field>
                                 <field name="engine">aggregation</field>
-                                <field name="formula">
-                                    (UG_TAX_D20.tax - UG_TAX_F30.disallowed)
-                                    * (UG_TAX_C1.base + UG_TAX_C2.base + UG_TAX_C4.base + UG_TAX_C5.base + UG_TAX_C6.base + UG_TAX_C7.base)
-                                    / (UG_TAX_C1.base + UG_TAX_C2.base + UG_TAX_C3.base + UG_TAX_C4.base + UG_TAX_C5.base + UG_TAX_C6.base + UG_TAX_C7.base)
```

Comment:
Why do we need to factorize this one ? :thinking:  It's not in the form A + B / C (which would cause issues when ignoring 0 div if C is 0 but A isn't), so I think we don't need to change it

Reply:
The reason i added the ignore 0 div here is to not break the report because if we don't change this one there will be 0 div errors. I think that because this is a tax report it means the formula can't be changed so we don't really have a way to mitigate 0 div errors in this case. Also when filling out a tax return in real life and C in this formula would be 0, I have to verify this but i think you would just ignore this and fill the result in as 0

Reply:
The reason i added the ignore 0 div here is to not break the report because if we don't change this one there will be 0 div errors. I think that because this is a tax report it means the formula can't be changed so we don't really have a way to mitigate 0 div errors in this case. Also when filling out a tax return in real life and C in this formula would be 0, I have to verify this but i think you would just ignore this and fill the result in as 0

Reply:
Sure, I just don't get why not need to split the existing expression into multiple ones. Couldn't you just add ignore_zero_division as a subformula of the existing expression, without any other change here and on the other occurrence I pointed ?